assets/Dockerfile

```
@@ -1,16 +1,17 @@
 # Dockerfile for pysen-test
-FROM alpine:3.13.2
+FROM alpine:3.14.6
 
-COPY --from=python:3.6.13-alpine3.13 /usr/local/ /usr/local/
-COPY --from=python:3.7.10-alpine3.13 /usr/local/ /usr/local/
-COPY --from=python:3.8.8-alpine3.13 /usr/local/ /usr/local/
-COPY --from=python:3.9.2-alpine3.13 /usr/local/ /usr/local/
+COPY --from=python:3.6.13-alpine3.14 /usr/local/ /usr/local/
+COPY --from=python:3.7.10-alpine3.14 /usr/local/ /usr/local/
+COPY --from=python:3.8.8-alpine3.14 /usr/local/ /usr/local/
+COPY --from=python:3.9.2-alpine3.14 /usr/local/ /usr/local/
```

It seems those tags aren't available. I think it's okay to upgrade the patch version numbers as:
```suggestion
COPY --from=python:3.8.13-alpine3.14 /usr/local/ /usr/local/
COPY --from=python:3.9.12-alpine3.14 /usr/local/ /usr/local/
```